tozny/resource_realm_identity.go

```diff
@@ -155,3 +156,54 @@ func resourceRealmIdentityDelete(ctx context.Context, d *schema.ResourceData, m
 
 	return diags
 }
+func resourceRealmIdentityUpdate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	var diags diag.Diagnostics
+	toznySDK, err := MakeToznySDK(d, m)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	// Check if Password has changed
+	if d.HasChange("password") {
+		// Get the old password
+		oldPassword, _ := d.GetChange("password")
+		realmName := d.Get("realm_name").(string)
+		username := d.Get("username").(string)
+		brokerTargetURL := d.Get("broker_target_url").(string)
+		emailExpiryMinutes := d.Get("recovery_email_ttl").(int)
+		identityID, err := strconv.ParseInt(d.Id(), 10, 64)
+		if err != nil {
+			diag.FromErr(fmt.Errorf("Error unable to parse identity id: %+v %+v", d.Id(), err))
+		}
+		// Login the Identity
+		request := e3db.TozIDLoginRequest{
+			Username:   username,
+			Password:   oldPassword.(string),
+			RealmName:  realmName,
+			APIBaseURL: toznySDK.APIEndpoint,
+		}
+		identitySdk, err := e3db.GetSDKV3ForTozIDUser(request)
+		if err != nil {
+			diag.FromErr(fmt.Errorf("Error Could not log in %+v", err))
+		}
+		// Populate the Identity
+		identity := e3db.Identity{
+			ID:       identityID,
+			Username: username,
+			Realm: &e3db.Realm{
+				Name:               realmName,
+				App:                "account",
```

Comment:
no magic variables, this should come from an intention revealingly named constant